eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -608,3 +631,10 @@ jobs:
       artifact: $(Agent.JobName)_Artifacts$(artifactsSuffix)
       displayName: Publish Artifacts
       continueOnError: true
+
+    - task: PublishBuildArtifacts@1
```

Comment:
I think you're using PublishBuildArtifacts here to enable merging of manifests into a single artifact. Can you add a comment for that if so?

Reply:
yes, added